lib/modelState.py

```diff
@@ -12,6 +12,19 @@ class GPTState:
     context: ClassVar[list[GPTMessageItem]] = []
     thread: ClassVar[list[GPTMessage]] = []
     thread_enabled: ClassVar[bool] = False
+    debug_enabled: ClassVar[bool] = False
+
+    @classmethod
+    def enable_debug(cls):
+        """Enable debug printing"""
+        cls.context = []
+        actions.app.notify("Enabled debug logging")
+
+    @classmethod
+    def disable_debug(cls):
+        """Disable debug printing"""
+        cls.context = []
+        actions.app.notify("Disabled debug logging")
```

Comment:
Now that I am looking at this again, where is `debug_enabled` being toggled? Don't we also want to keep context the same? I think implicitly clearing it is confusing if the user wants to debug

Reply:
Oops that is a mistake-will fix